app/MMB/src/navigation/TravelDocumentModalScreen.js

```diff
@@ -63,24 +76,70 @@ export default class TravelDocumentModalScreen extends React.Component<
         </HeaderTitle>
       ),
       headerRight: (
-        <HeaderButton onPress={goBack}>
+        <HeaderButton
+          onPress={onSave}
+          disabled={navigation.state.params.disabled}
+        >
           <Translation id="mmb.missing_informaiton.travel_document_modal_screen.save" />
         </HeaderButton>
       ),
       gesturesEnabled: false,
     };
   };
 
+  componentDidMount = () => {
+    this.props.navigation.setParams({ onSave: this.onSave, disabled: true });
+  };
+
   onIdNumberChange = (idNumber: string) => {
-    this.setState({ idNumber });
+    this.setState(
+      state => ({
+        idNumber,
+        error: {
+          ...state.error,
+          idNumber: idNumber.length < 5 && idNumber.length !== 0,
```

Comment:
I believe it would be clearer to write: 
```
idNumber.length > 0 && idNumber.length < 5
```

Reply:
Also, is there a place where we would centralize these form-checking conditions?

Reply:
yes, thank you, that looks better